packages/tools/piw-utils-internal/src/widgets/FilterProvider.ts

```diff
@@ -2,13 +2,16 @@ import { Context, Dispatch } from "react";
 import { ListAttributeValue } from "mendix";
 import { FilterCondition } from "mendix/filters";
 
+export declare type FilterValue = { type: string; value: any };
```

Comment:
This is a utility file. I think it's good to move it to the utils folder.

Reply:
This is specific for filters and is already inside piw-utils-internals

Reply:
Yes, that's what I see as well. Wouldn't it be more suitable to place it in `packages/tools/piw-utils-internal/src/utils`? This file doesn't contain a widget/component.

Reply:
Done